cspell.yaml

```diff
@@ -158,8 +159,8 @@ words:
   - wlroots
   - wlsunset
   - wofi
-  - wttr
   - Wttr
```

Comment:
looks like the tools we used to sort the list disagree about the order of capitalized words. I just checked though and we can remove `Wttr` since `wttr` is there too.

Reply:
I used nvim's `:sort i`.

> I just checked though and we can remove Wttr since wttr is there too.

Awesome.